Library/Homebrew/diagnostic.rb

```diff
@@ -274,11 +274,11 @@ def check_for_stray_pcs
 
       def check_for_stray_las
         allow_list = [
-          "libfuse.la", # MacFuse
-          "libfuse_ino64.la", # MacFuse
-          "libosxfuse_i32.la", # OSXFuse
-          "libosxfuse_i64.la", # OSXFuse
-          "libosxfuse.la", # OSXFuse
+          "libfuse.la", # macFUSE
+          "libfuse_ino64.la", # macFUSE
+          "libmacfuse_i32.la", # macFUSE
+          "libmacfuse_i64.la", # macFUSE
+          "libmacfuse.la", # macFUSE
```

Comment:
Add these to the existing list instead.

Reply:
I checked around for an _existing list_, however, didn't find anything apparent. Could you provide a little more context as to what you're referring to? Thanks!

Reply:
You're changing `libosxfuse` to `libmacfuse` inside of an array. Instead of doing that, just add extra `libmacfuse` lines so this retains compatibility with older osxfuse installs.